_posts/platform/app/2000-01-01-autoscaler.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Scale automatically your application
+nav: Autoscaling
+modified_at: 2018-04-05 00:00:00
+tags: app scaling metrics autoscaler
+---
+
+{% note %}
+  You might want to read first the page about [scaling an application]({% post_url
+  platform/app/2000-01-01-scaling %}).
+{% endnote %}
+
+The autoscaling feature lets you scale automatically your application up and down, depending on a
+user-defined threshold on the application metric.
+
+An autoscaler can be added to an application by going to the "Containers" tab of your app:
+
+{% assign img_url = 'https://cdn.scalingo.com/documentation/screenshot_dashboard_autoscalers.png' %}
+{% include mdl_img.html %}
+
+The **Target** is used by the autoscaler to adapt the number of containers of your application. We
+also provide you with the 95th percentile average over the last 24 hours as a recommended value.
```

Comment:
I don't get it, what's a 95th percentile average? It sounds like two different things to me.